payroll/apps.py

```diff
@@ -46,4 +48,17 @@ def __load_config(cls, cfg):
         CustomFilterRegistryPoint.register_custom_filters(
             module_name=cls.name,
             custom_filter_class_list=[BenefitPlanCustomFilterWizard]
-        )
\ No newline at end of file
+        )
+
+        from payroll.strategies import (
+            StrategyOnlinePayment,
+            StrategyMobilePayment,
+            StrategyOnSitePayment
+        )
+        PaymentsMethodRegistryPoint.register_payment_method(
+            payment_method_class_list=[
+                StrategyOnlinePayment(),
+                StrategyMobilePayment(),
+                StrategyOnSitePayment()
+            ]
+        )
```

Comment:
I'm not sure why there is extra code in __load_config method as it's only purpose is to load the config keys. I think this code (and the custom filters one) Should be in ready() method or in a new method.

Reply:
Moved to separate method which is triggered in 'ready' method after another steps